hydrabus_framework/core/logger.py

```diff
@@ -0,0 +1,44 @@
+from tabulate import tabulate
+
+
+class Logger:
+    def __init__(self):
+        self.HEADER = '\033[95m'
+        self.OKBLUE = '\033[94m'
+        self.OKGREEN = '\033[92m'
+        self.WARNING = '\033[93m'
+        self.FAIL = '\033[91m'
+        self.ENDC = '\033[0m'
+        self.BOLD = '\033[1m'
+        self.UNDERLINE = '\033[4m'
+
+    def print(self, text, category):
```

Comment:
Should use an enum instead of a string

Reply:
Done [#10c3681](https://github.com/hydrabus-framework/framework/commit/10c368189272dfb79bbaec95279da987552428a0)

Reply:
This point has been improved but not resolved. Please use enum, you will save stack spaces

Reply:
Done [#a9c2907](https://github.com/hydrabus-framework/framework/commit/a9c2907861118db4999c025149e6c6af6fc55198). I use an enum like method. I think the use of python enum doesn't fit in this case